src/test/java/org/opensearch/security/test/AbstractSecurityUnitTest.java

```diff
@@ -194,6 +195,17 @@ public void waitForInit(Client client) {
         }
     }
 
+    public static Settings.Builder nodeRolesSettings(final Settings.Builder settingsBuilder, final boolean isClusterManager, final boolean isDataNode) {
+        final ImmutableList.Builder<String> nodeRolesBuilder = ImmutableList.<String>builder();
+        if (isDataNode) {
+            nodeRolesBuilder.add("data");
+        }
+        if (isClusterManager) {
+            nodeRolesBuilder.add("cluster_manager");
```

Comment:
Use `DiscoveryNodeRole.CLUSTER_MANAGER_ROLE.roleName()`?

Reply:
There is no constant inside of OpenSearch for the config key which is the most obscure of these values, `node.roles`.  So I'm throwing my hands in the air?  IMO this is a good topic to revisit if we clean up the test construction workflow

Reply:
I believe `NodeRoleSettings.NODE_ROLES_SETTING.getKey()` resolves to `node.roles`.

Reply:
My last attempt to push back - I do not think it's worthwhile.  If you feel strongly, we could create an issue to clean this up or ask again and I'll just do it.

Reply:
I think it's fine as is. For my own understanding I searched through core to find where a list of the potential values was coming from. If someone comes to modify this function in the future to add additional node types they will need to be aware of what the list of potential node types is.

Reply:
>If someone comes to modify this function in the future to add additional node types they will need to be aware of what the list of potential node types is.

@peternied Are there any other occurrences of this? If so, we can resolve this in a separate issue for clean-up. Otherwise, could we avoid unnecessary follow-up issues only for this minor change? IMHO, @cwperks is making a legitimate request for this.

Reply:
> If someone comes to modify this function in the future to add additional node types they will need to be aware of what the list of potential node types is.

This is true. Using `DiscoveryNodeRole.CLUSTER_MANAGER_ROLE.roleName()` instead would make people easier to find out the list of potential node types when this function is getting changed in the future.

Reply:
@peternied Something to consider about using the existing constant here: `CLUSTER_MANAGER_ROLE` is a good example since it was recently renamed. If core modifies or removes a role then we will get a compilation error when using the constant. When hardcoding a string the code will still compile.

Reply:
It's been updated.

Reply:
Thank you @peternied!